yubikey-luks-enroll

```diff
@@ -93,4 +93,21 @@ else
 		if [ "$DBG" = "1" ]; then echo "LUKS key: $R"; fi
 fi
 
+# add keyscript to /etc/crypttab
+
+if [[ -b ${DISK} ]]
+then
+   echo "${DISK} searching from disk/by-uuid"
+   UUID="$( ls -l /dev/disk/by-uuid/ | grep $( basename ${DISK} ) | awk '{ print $9 }' )"
+else
+   echo "${DISK} searching from crypttab"
+   eval $( grep ${DISK} /etc/crypttab | awk '{ print $2 }' )
+fi
+
+if ! grep 'keyscript=/usr/share/yubikey-luks/ykluks-keyscript' /etc/crypttab
+then
+   sed -i "/${UUID}/ s/$/,keyscript=\/usr\/share\/yubikey-luks\/ykluks-keyscript/" /etc/crypttab
```

Comment:
Unfortunately this breaks for me. I think this is not very robust. Even in another shell (bash), this starts to break.